mapit_gb/management/commands/mapit_UK_show_missing_codes.py

```diff
@@ -30,18 +46,25 @@ def handle_noargs(self, **options):
                 type=area_type
             )
             for code_type in used_code_types:
-                areas_without_codes = areas.exclude(codes__type__code=code_type)
-                if areas_without_codes.count() > 0:
-                    print '  {count} {type} areas have no {code_type} code:'.format(
-                        count=areas_without_codes.count(),
-                        type=area_type,
-                        code_type=code_type
-                    )
-                    for area in areas_without_codes:
-                        print '    {name} {id} (gen {gen_low}-{gen_high}) {country}'.format(
-                            name=area.name,
-                            id=area.id,
-                            gen_low=area.generation_low_id,
-                            gen_high=area.generation_high_id,
-                            country=area.country
+                if self.is_code_type_required_for_area_type(code_type, area_type):
```

Comment:
I tend to prefer to return early in cases like this in Python, because it reduces the indentation of the rest of the method and makes it clear that there's no `else` - we don't do anything when the condition is false. It would also make the diff in this commit smaller 🔻